app/views/layouts/application.html.erb

```diff
@@ -77,7 +77,7 @@
                   <span class="caret"></span>
                 </a>
                 <ul class="dropdown-menu" role="menu">
-                  <% I18n.available_locales.each do |locale| %>
+                  <% [:en, :pt].each do |locale| %>
```

Comment:
I was changed before ```devise-i18n``` was removed from Gemfile. We need to undo this modification.